RELEASE_NOTES.md

```diff
@@ -13,6 +13,13 @@ Release notes template:
 ## Removed
 
 -->
+# 2019-08-05
+
+## Changed
+
+* Music Reserves import now appends courses for recordings instead of simply
+skipping them.
```

Comment:
probably should clarify "existing" recordings

Reply:
Updated.